tests/functional/globals.py

```diff
@@ -1,13 +1,27 @@
 import os
 
+def get_builtin_modules():
+    module_path = os.popen("%s --version | grep \"Module Path\" | cut -d ' ' -f 3" % get_syslog_ng_binary(), 'r').read().split(".")
+    return '%s:' % module_path
+
```

Comment:
Okay sorry yes that's true. I fixed the names based on your  instructions. From now on I guess it is clear which functions would do what.